src/main/java/org/roda_project/commons_ip2/model/impl/eark/EARKSIP.java

```diff
@@ -227,7 +227,7 @@ private Path getZipPath(Path destinationDirectory, String fileNameWithoutExtensi
     if (fileNameWithoutExtension != null) {
       zipPath = destinationDirectory.resolve(fileNameWithoutExtension + SIP_FILE_EXTENSION);
     } else {
-      zipPath = destinationDirectory.resolve(getId().replaceAll("[^a-zA-Z0-9-_\\.:]", "_") + SIP_FILE_EXTENSION);
+      zipPath = destinationDirectory.resolve(getId().replaceAll("[^a-zA-Z0-9-_\\.]", "_") + SIP_FILE_EXTENSION);
```

Comment:
Backslashes are still allowed after this commit, the `\\` needs to be removed as well to fit your commit message and to generate (Windows-)safe directory names.